packages/react-scripts/template/gitignore

```diff
@@ -0,0 +1,79 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
shouldnt it be .gitignore?

Reply:
it is later renamed to .gitignore
this is their solution to .gitignore being used as npmignore